matRadGUI.m

```diff
@@ -1748,7 +1748,6 @@ function btnuiTableAdd_Callback(hObject, ~, handles)
 data{sEnd+1,2} = 'Select VOI Type';
 data{sEnd+1,3} = 2;
 data{sEnd+1,4} = 'Select obj func/constraint';
-data{sEnd+1,6} = '';
```

Comment:
Really good catch. But wouldn't it more consistent to initiale an empty Array with[]?